.github/workflows/runnable_cxx.yml

```
@@ -138,7 +135,7 @@ jobs:
       with:
           # The host compiler does not matter too much, it is simply used for building DMD
           # Other CIs will test bootstrapping.
-          compiler: dmd-2.091.0
+          compiler: dmd-2.095.1
```

Maybe use ldc as the host compiler (as mentioned in the issue)?

```suggestion
          compiler: ldc-1.25.1
```

---

Then the generated DMD won't segv, but the test suite will.